.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ocaml-github"]
+	path = ocaml-github
+	url = https://github.com/mirage/ocaml-github.git
```

Comment:
To get CI to pass, you'll also need to mark this as vendored. In the `dune` file, add:
```
(vendored_dirs ocaml-github)
```